tests/requirements-libraries.txt

```diff
@@ -52,5 +49,17 @@ matplotlib==3.3.4
 
 # iPython 7.17 dropped support for python 3.6
 # https://ipython.readthedocs.io/en/stable/whatsnew/version7.html#ipython-7-17
-ipython==7.19.0; python_version > '3.6'
-ipython==7.16.1; python_version == '3.6'  # pyup: ignore
+ipython==7.20.0; python_version > '3.6'
+ipython==7.16.1; python_version < '3.6'  # pyup: ignore
+
+# pandas also dropped support
+pandas==1.2.2; python_version > '3.6'
+pandas==1.5.4; python_version < '3.6'  # pyup: ignore
```

Comment:
Shouldn't this be pandas==1.1.5 for python 3.6?

Reply:
And in addition, python 3.6 is slipping through because the python version conditions are strictly greater and strictly lower... (that's why CI is not complaining about invalid pandas version).

Reply:
Fixed, thanks.